packages/builtin/numeric_array_converter.pony

```diff
@@ -0,0 +1,13 @@
+primitive NumericArrayConverter
```

Comment:
This doesn't seem to be required for the `buffered` perf improvements, and as a new feature it should probably go through the RFC process (or, at the very least, be in a separate PR).